pacta/README.md

```diff
@@ -29,24 +31,28 @@ Run the build_with_tag.sh script from this directory, specifying a tag to assign
 The script will:
 - clone the repos locally, only copying the current version of the files
 - remove any existing "2dii_pacta" images from your local docker system
-- build a "2dii_pacta" docker image using the Dockerfile in this directory, which will
+- build a "2dii_pacta" docker image using the Dockerfile in this directory,
+which will
   - use 2dii/r-packages as a base
   - copy in the frshly cloned repos
   - make some necessary permissions changes
-- use pacta_web_template.zip as a template to create new zip file named pacta_web.zip in this directory which will contain
   - an export of the freshly made docker image gzipped (2dii_pacta.tar.gz)
```

Comment:
start this point with a verb, e.g. "export the..." or "make an export of the..."